Map.py

```diff
@@ -21,19 +21,19 @@ def __initializeMapContent(self, mapContent: jsonDict) -> None:
         :param base: A dictionary containing the map content for the map.
         '''
         for baseHex in mapContent["base"]:
-            self.__map[HexToTuple(baseHex)] = "Base"
```

Comment:
Base, Obstacle ... it can be enum like HexType

Reply:
added for both Hex and Tank types